blocks/library/list/index.js

```diff
@@ -2,23 +2,21 @@
  * Internal dependencies
  */
 import './style.scss';
+import { registerBlock, query } from 'api';
+import Editable from 'components/editable';
 
-const Editable = wp.blocks.Editable;
-const { html, prop } = wp.blocks.query;
+const { html, prop } = query;
```

Comment:
That we can't destructure `query` here is a little disappointing. Might be worth a rethink on exporting `hpq` as `wp.blocks.query`, either renaming the property or simply assuming dependent plugins refer to hpq directly.